docs/proxies.rst

```diff
@@ -71,3 +71,12 @@ When you implement proxies, the best documentation to follow is the source of To
 
 Sometimes precondition checks are impossible because the specified block is too old (pruned in the Ethereum client).
 In this case, the precondition check can be skipped.
+
+When you implement a new method of a proxy, consider refusing the string ``"last"`` as the block identifier.
```

Comment:
`last` is not a valid value, the possible tags are `latest` `pending` or `earliest`. I believe here you meant `pending`

Reply:
I meant `latest`.  I replaced strings.